src/core/Rational.pm6

```diff
@@ -210,16 +217,18 @@ my role Rational[::NuT = Int, ::DeT = ::("NuT")] does Real {
             }
         }
         else {
-            $prec = ($!denominator < $base**6 ?? 6 !! $!denominator.log($base).ceiling + 1);
+            my $lim = 10**307;
```

Comment:
Could you add a comment about where `10**307` comes from?

Reply:
Comments added